docs/snippets/common/main-config-framework-options-builder-use-swc.ts.mdx

```diff
@@ -14,11 +14,6 @@ const config: StorybookConfig = {
     },
   },
   stories: ['../src/**/*.mdx', '../src/**/*.stories.@(js|jsx|ts|tsx)'],
```

Comment:
I've been noticing a pattern of renaming snippets lately, and if you're ok, we'll need to figure out not only the naming convention and what goes in and what doesn't go in them